data/kea-dhcp/kea-dhcp6.conf

```diff
@@ -0,0 +1,48 @@
+{
+# DHCPv6 configuration starts on the next line
+"Dhcp6": {
+
+# First we set up global values
+    "valid-lifetime": 4000,
+    "renew-timer": 1000,
+    "rebind-timer": 2000,
+    "preferred-lifetime": 3000,
+
+# Next we set up the interfaces to be used by the server.
+    "interfaces-config": {
+        "interfaces": [ ]
+    },
+
+# And we specify the type of lease database
+    "lease-database": {
+        "type": "memfile",
+        "persist": true,
+        "name": "/var/lib/kea/dhcp6.leases"
+    },
+
+# Finally, we list the subnets from which we will be leasing addresses.
+    "subnet6": [
+        {
+            "subnet": "2001:db8:1::/64",
```

Comment:
`2001:db8::` is reserved for documentation and despite it SHOULD not be used elsewhere (or anywhere), it is also kinda not good to be used here.

If possible I would rather assign a other Site local private network, e.g. `fee1:ca1:1/64`.

This would apply to all places where `2010:db8` is used.

Reply:
I modified the private network in the code. I used the subnet fe12:3456:789a::/48.